docs/kyma-environment-broker/03-09-subaccount-cleanup-cronjob.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Subaccount Cleanup CronJob
+type: Details
+---
+
+Each SKR instance in KEB database belongs to global account and subaccount.
+Subaccount Cleanup is an application which periodically calls CIS service about SUBACCOUNT_DELETE events 
+and based on those events trigger de-provision action on SKR instance which subaccount belongs to.
+
+## Details
+
+All Subaccount Cleanup process is divided into several steps
+
+1. Fetch SUBACCOUNT_DELETE events from CIS service
+
+    a. CIS client makes first call to CIS service, as a response it gets list of events  divided into pages, 
+       total number of events and total number of pages
+    b. CIS client fetches rest of the events by make call to each page one by one
+    c. a subaccount is taken from each event and keep in array
+    d. when CIS client ends, display logs with information how many subaccounts fetched
+
+2. Find all instances in KEB database based on subaccounts fetched in step 1
+
+    a. the whole subaccounts pool is divided into pieces
```

Comment:
I wouldn't divide it into separate points. Make a full sentence out of it and add it to point no.2:

```
The subaccounts pool is divided into pieces. For each piece, a query is made to the database to fetch instances.
```